### DISCUSSION TITLE
PAT Basic 1047. 编程团体赛 (20) (C语言实现) | PAT Solution in C

### DISCUSSION THREAD

**Discussion by xlucn** [xlucn/PAT#57] [General]
https://oliverlew.github.io/PAT/Basic/1047.html 

 题目

**Comment by XuHuang441** [comment]
有没有比o(n)更好的算法啊?

**Comment by xlucn** [comment]
> @AttitudeMatters
> 有没有比o(n)更好的算法啊?
> 

这要算总和的，不遍历一遍，应该做不到。更重要的是，这种不复杂的指令，远比读取快，所以O(N)的方法绝对不是瓶颈了。

**Comment by XuHuang441** [comment]
> @OliverLew
> > @AttitudeMatters
> > 有没有比o(n)更好的算法啊?
> > 
> 
> 这要算总和的，不遍历一遍，应该做不到。更重要的是，这种不复杂的指令，远比读取快，所以O(N)的方法绝对不是瓶颈了。

多谢大佬回答
